src/Features/CSharp/Portable/ImplementInterface/AbstractChangeImplementionCodeRefactoringProvider.cs

```diff
@@ -147,7 +147,7 @@ private static async Task<(SyntaxNode?, ExplicitInterfaceSpecifierSyntax?, Synta
 
         private static (SyntaxNode?, ExplicitInterfaceSpecifierSyntax?, SyntaxToken) GetContainer(SyntaxToken token)
         {
-            for (var node = token.Parent; node != null; node = node.Parent)
+            for (SyntaxNode? node = token.Parent; node != null; node = node.Parent)
```

Comment:
>SyntaxNode? [](start = 17, length = 11)

Is this change needed?

Reply:
Yes because of the `node = node.Parent` incrementor

---
In reply to: [362869806](https://github.com/dotnet/roslyn/pull/40699#discussion_r362869806) [](ancestors = 362869806)

Reply:
Once we remove w-warnings for `var` though this wouldn't need any changes.

---
In reply to: [362889717](https://github.com/dotnet/roslyn/pull/40699#discussion_r362889717) [](ancestors = 362889717,362869806)